Utils/azure-explorer-common/test/java/com/microsoft/tooling/msservices/serviceexplorer/azure/rediscache/RedisCacheCreateIntegrationTest.java

```diff
@@ -69,7 +70,7 @@
 @PowerMockIgnore("javax.net.ssl.*")
 @RunWith(PowerMockRunner.class)
 @PrepareForTest({ AuthMethodManager.class, AzureManager.class, SubscriptionManager.class, DefaultLoader.class })
-
+@Ignore
```

Comment:
Ignore this test for now because the requests/response records are out-of-date. Tracked by https://github.com/Microsoft/azure-tools-for-java/issues/1883